src/gui/dialogs/unit_recall.cpp

```diff
@@ -146,6 +146,14 @@ static std::string get_title_suffix(int side_num)
 	return msg.str();
 }
 
+void unit_recall:unit_recall_default_compare(unit &first, unit &second) {
```

Comment:
Oh, because you aren't mutating the units, it's better to receive them as const references: `const unit& first, const unit& second`